test/functional/api/cas/cli_messages.py

```diff
@@ -25,10 +25,26 @@
     r"Cache is in incomplete state - at least one core is inactive"
 ]
 
+add_cached_core = [
+    r"Error while adding core device to cache instance \d+",
+    r"Core device",
+    r"is already cached\."
+]
+
+remove_mounted_core = [
+    r"Can\'t remove core \d+ from cache \d+\. Device",
+    r"is mounted\!"
+]
+
+stop_cache_mounted_core = [
+    r"Can\'t stop cache instance \d+\. Device",
+    r"is mounted\!"
+]
+
```

Comment:
Do not split lines - instead add pattern for /dev/casX-Y (which would be /dev/cas\d+-\d+)

Reply:
In case add_cached_core there could be any disk or partition, so I can't replace it. Two others replaced.

Reply:
There are also patterns for that - for example \S+ matches one or more non-whitespace characters. so /dev/\S+ should match any device. Generally - don't split line here if it's one line in the output.